.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,27 @@
+name: linkcheck
+
+on:
+  pull_request:
+    paths:
+      - 'book/**'
```

Comment:
Do you mind adding a rule to run on `unstable`? It seems that we may as well run it slightly more often

Reply:
It makes sense, thank you!